test/test.txt

```diff
@@ -0,0 +1,3 @@
+fuck=you
```

Comment:
please reword the test file to not contain words that may offend people against anal sex